tools/issue-labeler/backfill.go

```diff
@@ -166,7 +166,9 @@ func updateIssues(issueUpdates []IssueUpdate, dryRun bool) {
 			}
 			var errResp ErrorResponse
 			json.Unmarshal(body, &errResp)
-			glog.Infof("API returned message: %s", errResp.Message)
+			if errResp.Message != "" {
+				glog.Infof("API returned message: %s", errResp.Message)
```

Comment:
Since this is logged as info, could we somehow make it more obvious this is an error. Perhaps just "API error: %s". Then you can search for "error" in the log.